core/domain/feedback_services.py

```diff
@@ -172,38 +244,58 @@ def get_next_page_of_all_feedback_messages(
     return (result_messages, new_urlsafe_start_cursor, more)
 
 def get_thread_analytics_multi(exploration_ids):
-    """Returns a dict with feedback thread analytics for the given exploration
-    ids.
+    """Fetch all FeedbackAnalytics for all the given exploration ids
+
+    Args:
+        exploration_ids: list of str. A list of exploration id
+
+    Returns:
+        list of FeedbackAnalytics
     """
-    return feedback_jobs_continuous.FeedbackAnalyticsAggregator.get_thread_analytics_multi( # pylint: disable=line-too-long
-        exploration_ids)
+    return feedback_jobs_continuous.FeedbackAnalyticsAggregator.\
+        get_thread_analytics_multi(exploration_ids)
 
 
 def get_thread_analytics(exploration_id):
-    """Returns a dict with feedback thread analytics for the given exploration.
+    """Fetch all FeedbackAnalytics for the given exploration id
 
-    The returned dict has two keys:
-    - 'num_open_threads': the number of open feedback threads for this
-        exploration.
-    - 'num_total_threads': the total number of feedback threads for this
-        exploration.
+    Args:
+        exploration_id: str.
+
+    Returns:
+        list of FeedbackAnalytics
     """
-    return feedback_jobs_continuous.FeedbackAnalyticsAggregator.get_thread_analytics( # pylint: disable=line-too-long
-        exploration_id)
+    return feedback_jobs_continuous.FeedbackAnalyticsAggregator.\
+        get_thread_analytics(exploration_id)
 
 
 def get_total_open_threads(feedback_thread_analytics):
-    """Returns the count of all open threads for the given
-    FeedbackThreadAnalytics domain objects."""
+    """Get the count of all open threads for the given FeedbackThreadAnalytics
+
+    Args:
+        feedback_thread_analytics: FeedbackThreadAnalytics
+
+    Returns:
+        int. The count of all open threads for the given FeedbackThreadAnalytics
+    """
     return sum(
         feedback.num_open_threads for feedback in feedback_thread_analytics)
 
 
 def create_suggestion(exploration_id, author_id, exploration_version,
                       state_name, description, suggestion_content):
-    """Creates a new SuggestionModel object and the corresponding
-    FeedbackThreadModel object."""
+    """Create a new SuggestionModel and the corresponding FeedbackThreadModel
 
+    Args:
+        exploration_id: str. The exploration id the suggestion belongs.
+        author_id: str. ID of the user who submitted the suggestion.
+        exploration_version: int or None. The exploration version for
+            which the suggestion was made.
+        state_name: str or None. The state name of the suggestion.
+        description: str. Learner-provided description of suggestion changes.
+        suggestion_content: dict. Key is the suggestion type while value is
```

Comment:
Hm, that doesn't sound right, I think. Can you described how you arrived at this conclusion?

Note that one thing you can do is trace the code to the controller layer, where you'll find, in core.controllers.feedback.SuggestionHandler, that it comes from the frontend. So then you can look in the frontend, in the player view. Take a look at, e.g., core/templates/dev/head/player/exploration_player.html -- you'll see the "Suggest a Change" modal, as well as submitSuggestion(). Do a grep for submitSuggestion -- e.g. `grep submitSuggestion . -r --exclude-dir=third_party` -- and you'll be able to find where it's defined, etc. Continue until you figure out where the suggestion_content dict comes from. (This is the sort of digging that's important to do when investigating a codebase; grep is your friend here.)

Then, to verify your answer, start up a local instance of the dev server, and in the backend, in this function, do something like `raise Exception(suggestion_content)`. While you have the dev server running, submit a suggestion via the dev server UI, and see what gets raised in the logs.

You'll probably find these two techniques pretty useful when developing on Oppia (or when developing in general), so I figured I'd mention them!